runners/samza/src/test/java/org/apache/beam/runners/samza/runtime/SamzaStoreStateInternalsTest.java

```diff
@@ -200,7 +199,6 @@ public void processElement(
     @Override
     public KeyValueStore<byte[], byte[]> getKVStore(
         String storeName,
-        TaskModel taskModel,
```

Comment:
We won't need this if we are on Samza 1.6

Reply:
this is only available in samza 1.7 which is not yet released